<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyQtClass</class>
 <widget class="QMainWindow" name="MyQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyQtClass</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

background-color: rgb(69, 137, 156);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="usertext">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>-10</y>
      <width>691</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bodoni MT Black</family>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>collage generator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="collagebtn_ver">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>491</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>создать по вертикали</string>
    </property>
   </widget>
   <widget class="QPushButton" name="collagebtn_hor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>491</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>создать по горизонтали</string>
    </property>
   </widget>
   <widget class="QPushButton" name="collagebtn_ver_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>621</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>в 2 столбца по вертикали</string>
    </property>
   </widget>
   <widget class="QPushButton" name="collagebtn_hor_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>621</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>в 2 строки по горизонтали</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
